Exception in thread "main" org.openqa.selenium.SessionNotCreatedException

```
jmusicbot  | [09:27:59] [INFO] [JMusicBot]: Loaded config from /app/config.txt
jmusicbot  | Exception in thread "main" org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
jmusicbot  | Host info: host: '21935414a705', ip: '10.69.0.64'
jmusicbot  | 	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:563)
jmusicbot  | 	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:245)
jmusicbot  | 	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)
jmusicbot  | 	at org.openqa.selenium.chromium.ChromiumDriver.<init>(ChromiumDriver.java:114)
jmusicbot  | 	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:88)
jmusicbot  | 	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:83)
jmusicbot  | 	at com.jagrosh.jmusicbot.utils.YouTubeUtil.GetPoTokenAndVisitorData(YouTubeUtil.java:117)
jmusicbot  | 	at com.jagrosh.jmusicbot.JMusicBot.startBot(JMusicBot.java:172)
jmusicbot  | 	at com.jagrosh.jmusicbot.JMusicBot.main(JMusicBot.java:70)
jmusicbot  | Caused by: org.openqa.selenium.WebDriverException: Driver server process died prematurely.
jmusicbot  | Build info: version: '4.27.0', revision: 'd6e718d134'
jmusicbot  | System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-130-generic', java.version: '11.0.25'
jmusicbot  | Driver info: driver.version: ChromeDriver
jmusicbot  | 	at org.openqa.selenium.remote.service.DriverService.start(DriverService.java:246)
jmusicbot  | 	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:117)
jmusicbot  | 	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
jmusicbot  | 	... 8 more
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,18 +1,17 @@
-FROM eclipse-temurin:11-jre
-ARG DL_URL
-ADD --chmod=755 ${DL_URL} /app/JMusicBot.jar
-COPY --chmod=755 entrypoint.sh /entrypoint.sh
-RUN \
-    apt update && \
-    apt install -y --no-install-recommends \
+FROM eclipse-temurin:21-jre-alpine
+RUN addgroup -g 1000 user && \
+    adduser -D -u 1000 -G user user && \
+    mkdir /app && \
+    chown user:user /app && \
+    apk add --no-cache \
         bash \
         chromium \
-        chromium-chromedriver && \
-    apt clean && \
-    rm -rf \
-        /var/lib/apt/lists/* \
-        /tmp/* \
-        /usr/share/doc \
-        /usr/share/man
+        chromium-chromedriver
+
+USER user:user
+ARG JMB=https://github.com/cwlu2001/MusicBot/releases/download/0.4.4-dev.5/JMusicBot-0.4.4-dev.5.jar
+ADD --chmod=755 ${JMB} /app/JMusicBot.jar
+COPY --chmod=755 entrypoint.sh /entrypoint.sh
+ENV CHROMIUM_USER_FLAGS=--no-sandbox
 WORKDIR /app/config
 ENTRYPOINT [ "/entrypoint.sh" ]

